ssl/statem/statem_lib.c

```diff
@@ -278,10 +278,10 @@ MSG_PROCESS_RETURN tls_process_cert_verify(SSL *s, PACKET *pkt)
     EVP_PKEY *pkey = NULL;
     const unsigned char *data;
 #ifndef OPENSSL_NO_GOST
-    unsigned char *gost_data = NULL;
+    unsigned char *gost_data = NULL;
```

Comment:
Rewriting spaces. (So silly mistake. Not sure if it's caused by my online-editing.)